lib/ts/recipe/passwordless/index.ts

```diff
@@ -72,42 +72,12 @@ export default class RecipeWrapper {
             userContext: normalisedUserContext,
         });
 
-        if (previousAttemptInfo === undefined) {
-            throw new Error(
-                "No information for the previous attempt found, " +
-                    "createCode must be called once before trying to resend"
-            );
-        }
-
-        const resendCodeResponse = await recipe.recipeImplementation.resendCode({
+        return recipe.recipeImplementation.resendCode({
             ...input,
             userContext: normalisedUserContext,
-            deviceId: previousAttemptInfo.deviceId,
-            preAuthSessionId: previousAttemptInfo.preAuthSessionId,
+            deviceId: previousAttemptInfo === undefined ? "" : previousAttemptInfo.deviceId,
+            preAuthSessionId: previousAttemptInfo === undefined ? "" : previousAttemptInfo.preAuthSessionId,
```

Comment:
Add a comment as to why are are making them empty strings in case `previousAttemptInfo === undefined`.